### PR TITLE
Adds new design for left menu

### PR DESCRIPTION
**Pull request by Iloveall**
## Description

Fixes #...

## Notes (out of scope, known isues, hints for reviewing code, ...)  (optional)

- Changes left menu design
- Changes score score-ring, skill-progress, loading, error components
- Removes unused css variables

Left:

- Animation for loader
- Remove unused inputs for loading, skill-progress, score-ring, error components
- Empty chapter message
- Children loading
- New button in error component

## Test cases

- [ ] Case 1:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/new-design/layout-left-menu-tree/en/a/home;pa=0)
  3. Then I see new tree nav

### CODE REVIEW

File: src/app/core/components/left-nav-tree/left-nav-tree.component.ts [hunk 1]
```diff
@@ -19,6 +19,10 @@ export class LeftNavTreeComponent implements OnChanges {
   }
 
   private mapItemToNodes(data: NavTreeData): TreeNode<NavTreeElement>[] {
+    const inL1 = data.elements.some(
+      e => !data.selectedElementId || e.hasChildren
+        && (e.route.id === data.selectedElementId || e.children?.find(c => c.route.id === data.selectedElementId))
+    );
```
Comment (smadbe) [inline src/app/core/components/left-nav-tree/left-nav-tree.component.ts:25]: This check is a bit weird and no always valid I think. The cleanest way to know if we are on l1 is to add `onL1` boolean parameter to ` mapItemToNodes()` and to pass `true` when called on L18 and pass `false` on line 32.

**Review by smadbe — APPROVED** [review]

File: src/app/core/components/left-menu/left-menu.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
 <alg-left-header></alg-left-header>
-<perfect-scrollbar class="left-nav-theme scrollbar" [ngClass]="{'dark': isNavThemeDark}">
+<perfect-scrollbar>
   <alg-left-nav (themeChange)="onNavThemeChange($event.isDark)" (selectElement)="onSelectedElementChange($event)"></alg-left-nav>
```
[the remark this answers is not in the document]
Comment (Iloveall) [reply src/app/core/components/left-menu/left-menu.component.html:3]: Fixed #1365

File: src/app/core/components/left-nav-tree/left-nav-tree.component.html [hunk 3]
```diff
@@ -1,263 +1,256 @@
...
-        </span>
-        <span class="node-label">
-          <span class="label-container">
-            <span class="node-label-title">
-              {{ node.data.title }}
-              <span *ngIf="node.data.associatedGroupNames && node.data.associatedGroupNames.length > 0">
-                (<span>{{ node.data.associatedGroupNames.length | i18nPlural: { '=1': 'via group', 'other': 'via groups' } }}:</span>
-                {{ node.data.associatedGroupNames.join(', ') }})
-              </span>
-            </span>
```
[the remark this answers is not in the document]
Comment (Iloveall) [reply src/app/core/components/left-nav-tree/left-nav-tree.component.html:144]: Fixed #1362

File: src/app/core/components/left-nav-tree/left-nav-tree.component.html [hunk 4]
```diff
@@ -1,263 +1,256 @@
...
-          <i class="fa" [ngClass]="{ 'fa-folder': !node.expanded, 'fa-folder-open': node.expanded }"></i>
-        </span>
-        <span class="node-label">
-          <span class="label-container">
-            <span class="node-label-title">
-              {{ node.label }}
-              <span *ngIf="node.data.associatedGroupNames && node.data.associatedGroupNames.length > 0">
-                (<span>{{ node.data.associatedGroupNames.length | i18nPlural: { '=1': 'via group', 'other': 'via groups' } }}:</span>
-                {{ node.data.associatedGroupNames.join(', ') }})
-              </span>
```
[the remark this answers is not in the document]
Comment (Iloveall) [reply src/app/core/components/left-nav-tree/left-nav-tree.component.html:99]: Fixed #1362

File: src/app/core/components/left-nav-tree/left-nav-tree.component.ts (hunk 1 above)
Comment (Iloveall) [reply src/app/core/components/left-nav-tree/left-nav-tree.component.ts:25]: Fixed #1362

**Pull request merged**